src/modules/Dashboard/features/RespondentData/RespondentDataReview/ResponsesSummary/ResponsesSummary.tsx

```diff
@@ -14,13 +14,25 @@ export const ResponsesSummary = ({
 
   return (
     <StyledSummary data-testid={`${dataTestid}-description`}>
-      <Grid container columns={3} spacing={2.4}>
+      <Grid container columns={4} spacing={2.4}>
         {reviewDescription.map(({ id, title, content }) => (
           <Grid key={id} item xs={1}>
-            <StyledTitleMedium sx={{ color: variables.palette.on_surface_variant }}>
+            <StyledTitleMedium
+              sx={{
+                color: variables.palette.on_surface_variant,
+                fontWeight: variables.font.weight.bold,
+                fontSize: variables.font.size.sm,
+              }}
+            >
               {title}
             </StyledTitleMedium>
-            <StyledTitleBoldMedium sx={{ color: variables.palette.on_surface_variant }}>
+            <StyledTitleBoldMedium
+              sx={{
+                color: variables.palette.on_surface_variant,
+                fontWeight: variables.font.weight.regular,
+                fontSize: variables.font.size.md,
+              }}
+            >
```

Comment:
Nit: just wanted to confirm whether you checked that these styles aren't available as reusable styled components in [Typography.ts](https://github.com/ChildMindInstitute/mindlogger-admin/blob/develop/src/shared/styles/styledComponents/Typography.ts#L14)? Often predefined font styles from Figma have been defined there, or if they haven't, that's a good place to add it. Either way, not blocking. 🙂